plugins/modules/cp_gaia_routes_facts.py

```diff
@@ -98,7 +106,7 @@ def main():
     fields = dict(
         limit=dict(type="int", required=False, default=50),
         offset=dict(type="int", required=False, default=0),
-        order=dict(type="str", required=False, choices=['ASC', 'DESC'], default="ASC"),
+        order=dict(type="str", required=False, choices=['ASC', 'DESC'], default="ASC")
```

Comment:
maybe the comma in ansible is required after the parameter
please check it if it is working or return what was before